validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -32,11 +32,12 @@ class BlockPublisher(BlockPublisherInterface):
     other consensus algorithms
 
     """
-    def __init__(self, block_cache, state_view):
-        super().__init__(block_cache, state_view)
+    def __init__(self, block_cache, state_view, transaction_sender):
```

Comment:
dev_mode is the only working consensus in journal/consensus. The PoET consensus algorithm there and the implementation of this is still in progress, no implementation of the consensus interfaces are present yet.....